pkgs/data/fonts/curie/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchurl }:
+
+let
+  version = "1.0";
+in fetchurl rec {
+  name = "curie-${version}";
+
+  url = "https://github.com/NerdyPepper/curie/releases/download/v${version}/curie-v${version}.tar.gz";
+
+  downloadToTemp = true;
```

Comment:
Why is this required?

Reply:
without it I get `mkdir: cannot create directory '/nix/store/zxg3pw39mca775fzffg6jzvpzirpw0qm-curie-1.0': Not a directory`